src/Doctrine/EntityRelation.php

```diff
@@ -27,6 +27,7 @@ final class EntityRelation
     private bool $isSelfReferencing = false;
     private bool $orphanRemoval = false;
     private bool $mapInverseRelation = true;
+    private $joinTableName;
```

Comment:
```suggestion
    private ?string $joinTableName;
```

Reply:
I added a test case with a custom join table and added the above suggested change.